src/arm64/bitmanipulation.h

```diff
@@ -63,8 +64,9 @@ really_inline bool mul_overflow(uint64_t value1, uint64_t value2, uint64_t *resu
   return __builtin_umulll_overflow(value1, value2, (unsigned long long *)result);
 #endif
 }
+#endif // Currently unused
 
 } // namespace arm64
-} // namespace simdjson
+} // namespace {
```

Comment:
Though `namespace {` is fine, I think that the technical term might be "unnamed namespace". If you want to sound more scientific.

https://en.cppreference.com/w/cpp/language/namespace#Unnamed_namespaces